app/models/client.rb

```diff
@@ -35,6 +35,9 @@ def self.delegated_intake_attributes
 
   delegate :documents, to: :intake
   delegate *delegated_intake_attributes, to: :intake
+
+  scope :assigned_to, ->(user) { joins(:tax_returns).where({ tax_returns: { assigned_user_id: user } }) }
+
```

Comment:
This looks really useful. I'm also wondering if there's a way to set up a `has_many ... through: tax_returns` on the User model.